### ISSUE TITLE
iOS 9 Issue: Unexpected CFBundleExecutable Key

### ISSUE THREAD

**Issue by nowned** [wf9a5m75/phonegap-googlemaps-plugin#612]
Getting this issue when trying to build for iOS 9 with the newest XCode.

I've tried installing the newest version of the plugin to make sure.

![screen shot 2015-09-12 at 9 31 54 pm](https://cloud.githubusercontent.com/assets/1230091/9835258/a2c5db1c-5996-11e5-9695-e6e2ee42eb8d.jpg)

**Comment by Hirbod** [comment]
Disable bitcode, and remove the key. Check http://stackoverflow.com/questions/32096130/unexpected-cfbundleexecutable-key

**Comment by Hirbod** [comment]
Removing the plist entry for that specific problem will fix the issue. Google has fixed this in SDK Version 1.10.2, we're on 1.9.2. Will see how I can update this.

**Comment by Hirbod** [comment]
Updated Plugin to 1.3.0, which includes iOS SDK 1.10.2 
Please read README.

In order to install the new version, you have to:
```bash
1) Delete plugin 
> cordova plugin rm plugin.google.maps

2) Delete Google Maps iOS SDK 
> cordova plugin rm plugin.google.maps

3) Re-Install plugin from master (follow wiki for Mac/Linux/Windows) 
> cordova plugin add https://github.com/wf9a5m75/phonegap-googlemaps-plugin --variable API_KEY_FOR_ANDROID="YOUR_KEY" --variable API_KEY_FOR_IOS="YOUR_KEY"
```

**Comment by Mojo90** [comment]
FYI: While uploading the App to itunesconnect this Error is still in the newest plugin version. But just the ```Unexpected CFBundleExecutable Key```. Delete the entry fixed the problem....

**Comment by Hirbod** [comment]
This is a google issue, which was fixed in 1.11.1, which was released today. Will update tomorrow

**Comment by Hirbod** [comment]
Master will grab SDK 1.11.1 now - which will fix the problems.

```
$> cordova plugin rm plugin.google.maps
$> cordova plugin rm com.googlemaps.ios
reinstall...
```
